Source/CBL_BlobStore+Internal.h

```diff
@@ -0,0 +1,25 @@
+//
+//  CBL_BlobStore+Internal.h
+//  CouchbaseLite
+//
+//  Created by Jens Alfke on 8/23/15.
+//  Copyright © 2015 Couchbase, Inc. All rights reserved.
+//
+
+#import "CBL_BlobStore.h"
+
+
+// Name of file in blob-store dir that records encryption type used (currently "AES")
+#define kEncryptionMarkerFilename @"_encryption"
+
+
+@interface CBL_BlobStore ()
+
+- (NSString*) rawPathForKey: (CBLBlobKey)key;
+@property (readonly, nonatomic) NSString* tempDir;
+@property (readonly) CBLSymmetricKey* encryptionKey;
+
+#if DEBUG
+- (BOOL) markEncrypted: (BOOL)encrypted error: (NSError**)outError; // exposed for testing ONLY
+#endif
```

Comment:
I saw -markEncrypted: was used in regular codes to indicate encryption type in the blob store folder. I'm confused with #if DEBUG and the comment 'exposed for testing ONLY'. Should #if DEBUG be removed?